lib/ronin/cli/commands/archive.rb

```diff
@@ -42,15 +41,12 @@ module Commands
       class Archive < FileProcessorCommand
         usage '[options] [FILE ...]'
```

Comment:
Add space between the `class` line and the first code line, otherwise `rubocop` will complain.